internal/pkg/template/templates/environment/manifest.yml

```diff
@@ -79,3 +79,15 @@ http:
 observability:
   container_insights: {{.Observability.ContainerInsights}}
 {{- end}}
+
+# Configure additional permissions for the trust policy of the environment manager role to be able to assume it
+{{- if not .AdditionalAssumeRolePermissions}}
+# additionalAssumeRolePermissions:
+#   - sts:SetSourceIdentity
+#   - sts:TagSession
+{{- else}}
+additionalAssumeRolePermissions:
```

Comment:
This field name seems a little too specific. We usually have some level of abstraction for our manifest field because otherwise the experience will be almost the same as yaml override.

I would prefer not to add any additional field to the env manifest for now and only keep it as a flag at `env init`. Ideally we wouldn't need that flag either if we have a mechanism allowing users to customize their env bootstrap stack.

Reply:
> Ideally we wouldn't need that flag either if we have a mechanism allowing users to customize their env bootstrap stack.

With that in mind, I would agree that it would probably make a lot more sense to allow overrides for the bootstrap stack as well. Having a flag, which would change the stack only for one time wouldn't make much sense, as the changes would be overridden once the first env deployment is started.

Let me look into what would be needed to allow env overrides to kick in when bootstrapping an environment 👍

Reply:
I think it would take much more efforts for allowing env bootstrap override, so feel free to start from a flag if you think it's too overwhelming!

Reply:
Sorry for not coming back to you any sooner :(
I looked into the override stuff and from the concept point of view, I'm a bit stuck I think. YamlPatch would, iirc, error out, if there is an add/replace/remove statement for a yaml path, which does not exist. That means, that we couldn't easily re-use the existing overrides for environments, but would instead need to have an additional overrides location for the env bootstrap stack.
From my point of view this doesn't sound that nice at all, given that the overrides would only be used once during bootstrap. 
This issue does not exist in CDK, at least the user could workaround that by checking if a resource actually exists before doing things in CDK code.

Based on that, I would think that having a command line option only is probably the best way to go forward. Would you agree with this? 🤔